include/ros1_bridge/factory.hpp

```diff
@@ -33,6 +34,34 @@
 namespace ros1_bridge
 {
 
+static rclcpp::CallbackGroup::SharedPtr get_callback_group(
+  rclcpp::Node::SharedPtr ros2_node,
+  const std::string & topic_name = "")
+{
+  auto node_base = ros2_node->get_node_base_interface();
+  auto group = node_base->get_default_callback_group();
+  if (topic_name.empty()) {
+    // create a callback group with Reentrant for creating ros2 clients and services
```

Comment:
since this is a just bridge, it seems to be okay to have reentrant callback group for each client and service. that is the idea, correct? i just want to be sure if i understand this.

Reply:
Yes, that is what I understood.